draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,33 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update {#peer-address-update}
+
+When a datagram with a CID is received that has a source address different than the one currently associated with the DTLS connection, the receiver MUST NOT replace the source address currently stored in the association with the source address specified in the received datagram unless the following conditions are met:
+
+- The received datagram has been cryptographically verified using the DTLS record layer processing procedures.
+
+- The received datagram is "newer" (in terms of their epoch and sequence
+number) than the last datagram that successfully updated the source address.
+This condition ensures that replayed datagrams cannot be used to perform address 
+updates. Note that this condition does not help when the attacker is able to 
+replay datagrams that arrive earlier at the DTLS peer than the original datagram.
+
+- There is a strategy for performing a perform some reachability test. 
+No such test is defined in this	specification; {{!I-D.tschofenig-tls-dtls-rrc}} 
+defines an extension for such a DTLS protocol extension.
+
+The above ensures correctness of the protocol in presence of packet
+reordering at the network layer, while also thwarting a man-on-the-side
+attacker attempting to use spoofed or replayed records to reroute return
+traffic.
+
+Implementations that offer an application layer return routability check 
```

Comment:
```suggestion
Application protocols that implement protection against these attacks depend on
being aware of changes in peer addresses so that they can engage the necessary
mechanisms.
```

Then remove the remaining lines of the paragraph.

Reply:
The proposed changes are good for me